### ISSUE TITLE
[FEATURE] Drag & Drop connect CE without recognized relationship

### ISSUE THREAD

**Issue by ErHaWeb** [CodersCare/transfusion#7]
Currently, it is only possible to link content for which an (obvious/possible/broken/orphaned) link could be determined in the database in relation to a content element of the default language.

It would be nice if orphaned elements without a recognized reference to an element of the default language or elements for which the connections were explicitly removed via the wizard could also be placed in a new reference to each other using drag & drop.

![Bildschirmfoto 2024-03-18 um 13 25 27](https://github.com/CodersCare/transfusion/assets/5416710/24b051cf-6f21-4b3d-adad-4bc2043b4ef5)

Am I right in thinking that this feature is already part of the community budget idea for Q2?

**Comment by Bunnyfield** [comment]
Yes - Drag & Drop is one of the key features for Q2. We still need to consider different scenarios, where Drag & Drop might feel good at a first glance but might not be a good idea when taking a closer look. For example it would be counterproductive to freely rearrange connected records and assign them to different parents. 

Still for orphaned records it's definitely a good idea, but for the other connection levels the plan would be to enforce an orphaned record first by breaking the connection to be on the safe side. After that you could still drag the newly orphaned record to a new parent.
